### ISSUE TITLE
My own video is laggy in group calls

### ISSUE THREAD

**Issue by MarcAnt01** [UnigramDev/Unigram#2454]
**Describe the bug**
My own video is laggy in group calls (around 10 fps), anyway the other callers see the video properly and for me the videos of other callers aren't laggy. Basically the issue is the framerate of my own preview. CPU/GPU/Ram usage is all right.

**To Reproduce**
1. Make a call;
2. Share your camera;
3. Ask the other person to share the camera;
4. Enjoy the lags
5. 
**Expected behavior**
Have a smooth video also for myself

**Version Info**
 - Unigram: 7.8.6583
 - Windows: 11 Insider

**Comment by FrayxRulez** [comment]
Did you consider that this might be Windows 11 problem as you never reported this problem before apart from "When I share my screen, but it happens the same when I use any other screen recorder"?

**Comment by MarcAnt01** [comment]
> Did you consider that this might be Windows 11 problem as you never reported this problem before apart from "When I share my screen, but it happens the same when I use any other screen recorder"?

@FrayxRulez I thought this was due to the screen recording, but indeed it's not related at all.
Also Telegram Desktop seems to have a similar issue, but it's less noticeable (I think it's around 20/25 fps, while Unigram is 10/15).

**Comment by MarcAnt01** [comment]
Also for reference @FrayxRulez private calls don't have this issue, so it might be something related to the code of group calls

**Comment by FrayxRulez** [comment]
If it has started to happen on Windows 11 the problem is Windows 11.

**Comment by MarcAnt01** [comment]
Who said that it started happening with Windows 11?

**Comment by FrayxRulez** [comment]
You, as you reported this problem only after you updated to Windows 11.

**Comment by MarcAnt01** [comment]
No, I reported the problem as soon as I had the opportunity to make a long video call and not the usual tests to see if everything is working....

**Comment by MarcAnt01** [comment]
I'm 100% sure Windows 10 has the same issue

**Comment by FrayxRulez** [comment]
Anyway I won't fix this

**Comment by MarcAnt01** [comment]
Then, I see no point of using group videocalls in Unigram if I have to see my video lagging as hell

**Comment by FrayxRulez** [comment]
No one forces you to use Unigram.
